spec/diagnose_spec.rb

```diff
@@ -984,13 +994,23 @@
     matchers =
       case @runner.type
       when :ruby
-        { "gem_version" => VERSION_PATTERN }
+        {
+          "gem_version" => VERSION_PATTERN,
+          "name" => "DiagnoseTests"
+        }
+      when :elixir
+        {
+          "name" => "DiagnoseTests"
+        }
+      when :nodejs
+        {
+          "name" => be_empty.or(be_nil)
```

Comment:
Can you update the Node.js integration to send this param as well?

Reply:
Not sure? Unlike the others, it doesn't take a config file, so we'd have to pass it through environment variables.

Reply:
I think that would work? We can also set it the same for the other integrations if that makes it easier.